webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js

```diff
@@ -119,3 +119,20 @@ const createLocationItem = locations => {
   };
   return locItem;
 };
+
+export const onCollapse = () => {
+  document.getElementById('rails-app-content').classList.add('collapsed-nav');
```

Comment:
maybe we can update the class through the redux store changes instead + mounting the component with a new className?  same goes for `onExpand`